### ISSUE TITLE
Amoc not working with Erlang 19.0

### ISSUE THREAD

**Issue by ludwikbukowski** [esl/amoc#47]
Amoc version: 48140605ac0a3123fe36b6913382e74f71d2
Erlang version: **19.0**

``` bash
$ make rel
(...)
ERROR: OTP release 19 does not match required regex R?1[678]
ERROR: compile failed while processing /home/centos/load-testing/amoc/deps/goldrush: rebar_abort
make: *** [compile] Error 1
```
Amoc should support Erlang version 19.0 and above, what do you think?

**Comment by ludwikbukowski** [comment]
looks like branch `support-for-erlang-otp-19` is a cure for this issue

**Comment by s4kibs4mi** [comment]
Getting similar problem with erlang 19,

```
make[1]: Nothing to be done for `shared_libs'.
ERROR: OTP release 19 does not match required regex ^(R1[56]|17|18)
ERROR: compile failed while processing /Users/s4kib/HackLab/amoc/deps/exml: rebar_abort
make: *** [compile] Error 1
```

**Comment by studzien** [comment]
Resolved by #49
